pages/content/amp-dev/events/amp-roadshow.html

```diff
@@ -109,7 +109,9 @@ <h4>Want to add cities?
       {% endif %}
 
       <aside class="ap-o-cities-map-grid">
+        {% set next_cities = [] %}
         {% for place in doc.roadshow.nextstop if doc.roadshow.nextstop %}
```

Comment:
Just curious: can't those two loops be unified? In a way the one that's also printing the cities is checking if it's already printed?